tests/test_engine.py

```diff
@@ -972,38 +972,6 @@ def my_task(x):
         # Check that the task completed happily
         assert state.is_completed()
 
-    @flaky_on_windows
-    async def test_interrupt_task(self):
-        i = 0
-
-        @task()
-        def just_sleep():
-            nonlocal i
-            for i in range(100):  # Sleep for 10 seconds
-                time.sleep(0.1)
-
-        @flow
-        def my_flow():
-            with pytest.raises(TimeoutError):
-                with anyio.fail_after(1):
-                    just_sleep()
```

Comment:
It doesn't really make sense to have `anyio.fail_after` do anything in a synchronous context and this doesn't work in the new timeout implementation. This has been replaced with extensive tests in `tests/public/tasks/test_task_timeouts.py` that includes additional sync/async cases.